src/_scss/pages/generateDetachedFiles/detachedFileA.scss

```diff
@@ -71,6 +71,7 @@
             text-align: left;
             padding: rem(15);
             p {
+                max-width: none;
```

Comment:
I think this removed the sizing that we had before so now the detached file generation is extremely wide